operators/src/processing/expression/mod.rs

```diff
@@ -476,6 +474,8 @@ impl InitializedRasterOperator for InitializedExpression {
                     .boxed()
                 )
             }
+            // TODO: the zip method in the processor allows only a limited number of arrays. Use idx iterator and get_at_ instead?
```

Comment:
can we extend it to more inputs? We have to fix this somehow before merging.

Reply:
yes and no. we need to remove the number of zips. so we can either combine zips into tuples early or we can use the approach i used for the map stuff.